src/Storefront/Resources/views/storefront/element/cms-element-category-navigation.html.twig

```diff
@@ -1,10 +1,16 @@
-<div class="cms-element-{{ element.type }}">
-    {% set navigationResult1 = page.header.navigation.tree %}
-    {% set activeResult1 = page.header.navigation.active %}
-    <div class="category-navigation-box">
-        {% sw_include '@Storefront/storefront/layout/sidebar/category-navigation.html.twig' with {
-            navigationTree: page.header.navigation.tree,
-            activeResult: page.header.navigation.active
-        } only %}
+{% block element_category_navigation %}
+    <div class="cms-element-{{ element.type }}">
+        {% block element_category_navigation_element %}
+            {% set navigationResult1 = page.header.navigation.tree %}
+            {% set activeResult1 = page.header.navigation.active %}
```

Comment:
Can we remove that while you're at it? I don't think it's used anywhere

Reply:
I do indeed think that this is not used anywhere. But it might still be here for backwards compatibility for plugins that might use it?

Maybe a deprecation warning can be added here so it can be removed in the future? I'm not sure if that is possible for twig variables, as it is possible for fx. php classes and such.

Reply:
We branch off already 6.5.0.0, so we can't break anymore :)

Reply:
I just stumbled upon the same template and remembered this PR :-) 
But I think the variables can be safely removed, as the variables cannot be referenced anywhere. The current template has no blocks, so one can only overwrite it as whole, and the `sw_include` has the `only` attribute specified, hence one cannot not access these variables inside that template. It seems to me like two private unused variables (if I do not miss some twig trickery).

Reply:
I think your'e right @aragon999. I overlooked in the diff that this PR includes the block and there wasn't any blocks beforehand. With the block `element_category_navigation_box` someone could have used the variables inside of the block. But since there is no block and the include uses only I also cannot think of a way those variables could be used by 3rd-party code.

Reply:
Perfect :-) Fewer deprecations.